pkg/gcc/send_side_bwe.go

```diff
@@ -131,6 +136,14 @@ func (e *SendSideBWE) AddStream(info *interceptor.StreamInfo, writer interceptor
 
 // WriteRTCP adds some RTCP feedback to the bandwidth estimator
 func (e *SendSideBWE) WriteRTCP(pkts []rtcp.Packet, attributes interceptor.Attributes) error {
+	e.lock.RLock()
```

Comment:
I would suggest to use something like this here:
```
select {
    case <-e.close:
        return ErrSendSideBWEClosed
    default:
}
```
to save the boolean here.

Reply:
Thanks! I've updated the PR